setuper/review_change_proposal.py

```diff
@@ -24,7 +25,7 @@ def setup_review_change_proposal(account_name, iaso_client):
     ]
     org_unit_group_ids = [org_unit_group["id"] for org_unit_group in org_unit_groups]
     for org_unit in orgunits:
-        location = [None, {"latitude": org_unit["latitude"], "longitude": org_unit["longitude"], "altitude": 0.0}]
+        location = [None, {**org_unit_gps_point(org_unit)}]
```

Comment:
You will have `orgUnitId` as the first key of the dict. Will this work just like the old one with only `latitude`, `longitude` and `altitude`?